Library/Homebrew/test/cask/cmd/info_spec.rb

```diff
@@ -140,6 +142,9 @@
   end
 
   it "can run be run with a url twice", :needs_network do
+    analytics = '{"analytics":{"install":{"30d":{"docker": 1000},"90d":{"docker": 2000},"365d":{"docker": 3000}}}}'
+    expect(Utils::Analytics).to receive(:formulae_brew_sh_json).twice.with("cask/docker.json")
+    .and_return(JSON.parse(analytics))
```

Comment:
Instead of `JSON.parse` consider using a Ruby dictionary instead.

Reply:
done!